daq/runner.py

```diff
@@ -64,6 +67,7 @@ def __init__(self, config):
         self.exception = None
         self.run_count = 0
         self.run_limit = int(config.get('run_limit', 0))
+        self._default_port_toggle_timeout_sec = int(config.get('port_toggle_timeout_sec', 0))
```

Comment:
not sure "toggle" is the right word... this is for when the port is explicitly brought down, and then back up again, right?  Network term for that is to "flap" or "bounce" the port... port_flap_timeout_sec?